mockgcp/mockrun/testdata/job/crud/_http.log

```diff
@@ -0,0 +1,585 @@
+          "spec": {
+            "containers": [
+              {
+                "image": "us-docker.pkg.dev/cloudrun/container/job:latest",
+                "name": ""
+              }
+            ],
+            "nodeSelector": {}
+          }
+        }
+      }
+    }
+  },
+  "status": {
+    "latestCreatedExecution": {}
+  }
+}
+
+200 OK
+Content-Type: application/json; charset=UTF-8
+Server: ESF
+Vary: Origin
+Vary: X-Origin
+Vary: Referer
+X-Content-Type-Options: nosniff
+X-Frame-Options: SAMEORIGIN
+X-Xss-Protection: 0
+
+{
+  "apiVersion": "run.googleapis.com/v1",
+  "kind": "Job",
+  "metadata": {
+    "annotations": {
+      "run.googleapis.com/client-name": "gcloud",
+      "run.googleapis.com/client-version": "537.0.0",
+      "run.googleapis.com/creator": "yuwenma@google.com",
+      "run.googleapis.com/lastModifier": "yuwenma@google.com",
+      "run.googleapis.com/operation-id": "2f742195-f217-4dd5-bfac-0e257e4b3410"
+    },
+    "generation": 1,
+    "labels": {
+      "cloud.googleapis.com/location": "us-central1",
+      "run.googleapis.com/lastUpdatedTime": "2025-09-03T16:13:54.272555Z"
+    },
+    "name": "test-${uniqueId}",
+    "namespace": "${projectNumber}",
+    "selfLink": "/apis/run.googleapis.com/v1/namespaces/${projectNumber}/jobs/test-${uniqueId}"
+  },
+  "spec": {
+    "template": {
+      "metadata": {
+        "annotations": {
+          "run.googleapis.com/client-name": "gcloud",
+          "run.googleapis.com/client-version": "537.0.0",
+          "run.googleapis.com/execution-environment": "gen2"
+        },
+        "labels": {
+          "client.knative.dev/nonce": "buc_qig_tav"
+        }
+      },
+      "spec": {
+        "taskCount": 1,
+        "template": {
+          "spec": {
+            "containers": [
+              {
+                "image": "us-docker.pkg.dev/cloudrun/container/job:latest",
+                "resources": {
+                  "limits": {
+                    "cpu": "1000m",
+                    "memory": "512Mi"
+                  }
+                }
+              }
+            ],
+            "maxRetries": 3,
+            "serviceAccountName": "${projectNumber}-compute@developer.gserviceaccount.com",
+            "timeoutSeconds": "600"
+          }
+        }
+      }
+    }
+  },
+  "status": {}
+}
+
+---
+
+GET https://us-central1-run.googleapis.com/apis/run.googleapis.com/v1/namespaces/${projectId}/jobs/${jobID}?alt=json
+Accept: application/json
+Authorization: (removed)
+Connection: keep-alive
+
+200 OK
+Content-Type: application/json; charset=UTF-8
+Server: ESF
+Vary: Origin
+Vary: X-Origin
+Vary: Referer
+X-Content-Type-Options: nosniff
+X-Frame-Options: SAMEORIGIN
+X-Xss-Protection: 0
+
+{
+  "apiVersion": "run.googleapis.com/v1",
+  "kind": "Job",
+  "metadata": {
+    "annotations": {
+      "run.googleapis.com/client-name": "gcloud",
+      "run.googleapis.com/client-version": "537.0.0",
+      "run.googleapis.com/creator": "yuwenma@google.com",
```

Comment:
This should be normalized (probably operation-id and client-version also)

Reply:
This is the resource that has mismatch proto in gcloud (v1) and TF-based reconcilier (v2). I left the crud dir just for a record. I can delete the dir.